### PR TITLE
Selection_Sort console app.

### PR DESCRIPTION
**Pull request by SerhiiHarashchenko**
Please check my code.

### CODE REVIEW

File: Selection_Sort_C/main.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+
+void SortArray(int Array[], int sizeOfArray)
+{
+    int minElement = 0;
+    int thisElement = 0;
+    int helpElement = 0;
```
Comment (AntonLynnyk) [inline Selection_Sort_C/main.c]: tempElement

File: Selection_Sort_C/main.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    for(int i = 0; i < sizeOfArray - 1; i++)
+    {
+        minElement = i;
+
+        for(thisElement = i; thisElement < sizeOfArray; thisElement++)
+        {
+            if(Array[thisElement] < Array[minElement])
+            {
+                helpElement = Array[minElement];
```
Comment (AntonLynnyk) [inline Selection_Sort_C/main.c]: Please implement Swap(int *left, int *right)

File: Selection_Sort_C/main.c [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    for (int k = 0; k < sizeOfArray; k++)
+    {
+        printf("Sorted Array: %d\n", Array[k]);
+    }
+}
+
+int main()
+{
+    int UnsortedArray[] = {5, 14, 2, 1, -17, 4, 100, 832, -72, 44, 141};
```
Comment (AntonLynnyk) [inline Selection_Sort_C/main.c]: Please follow [camelCaseVariablesNaming ](https://techterms.com/definition/camelcase)

File: Selection_Sort_C/main.c [hunk 4]
```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+
+void SortArray(int Array[], int sizeOfArray)
```
Comment (AntonLynnyk) [inline Selection_Sort_C/main.c]: camelCase

File: Selection_Sort_C/main.c [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        for(thisElement = i; thisElement < sizeOfArray; thisElement++)
+        {
+            if(array[thisElement] < array[minElement])
+            {
+                Swap(&array[minElement], &array[thisElement]);
+            }
+        }
+    }
+
+    printf("Sorted array:\n");
```
Comment (AntonLynnyk) [inline Selection_Sort_C/main.c]: Please create a separate function(In C we call them functions) for printing the array.
Please notice, that your method is called SortArray and not SortAndPrintArray, so it shouldn't print anything.
When you implement method PrintArray, please call it from the Main method, after you sorted the array.

Comment (SerhiiHarashchenko) [reply Selection_Sort_C/main.c]: Done.

File: Selection_Sort_C/main.c [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+            {
+                Swap(&array[minElement], &array[thisElement]);
+            }
+        }
+    }
+}
+
+int main()
+{
+    int unsortedArray[] = {5, 14, 2, 1, -17, 4, 100, 832, -72, 44, 141};
```
Comment (AntonLynnyk) [inline Selection_Sort_C/main.c]: Looks like the name `unsortedArray` doesn't make much meaning, especially when you call PrintArray(unsortedArrary). It looks like you print unosortedArray, but it is now sorted. Please change name to something more general, like just `array`

File: Selection_Sort_C/main.c [hunk 7]
```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+
+void PrintArray(int array[], int sizeOfArray)
+{
+    printf("Sorted array: \n");
```
Comment (AntonLynnyk) [inline Selection_Sort_C/main.c]: You don't know here if it is sorted or not. This method just prints the array. Please remove the word `Sorted` from `printf`.

File: Selection_Sort_C/main.c [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+int main()
+{
+    int unsortedArray[] = {5, 14, 2, 1, -17, 4, 100, 832, -72, 44, 141};
+
+    int sizeOfArray = sizeof(unsortedArray) / sizeof(unsortedArray[0]);
+
+    SortArray(unsortedArray, sizeOfArray);
+
```
Comment (AntonLynnyk) [inline Selection_Sort_C/main.c]: It is hard to explain, but this new line isn't needed here. SortArray and PrintArray look like one logical block. It is better to group them together actually.

Comment (SerhiiHarashchenko) [reply Selection_Sort_C/main.c]: Delited line and changed name of an array.

File: Selection_Sort_C/main.c [hunk 9]
```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+
+void PrintArray(int array[], int sizeOfArray)
+{
+    printf("Sorted array: \n");
```
Comment (AntonLynnyk) [inline Selection_Sort_C/main.c]: GARRRRRRR!

Comment (SerhiiHarashchenko) [reply Selection_Sort_C/main.c]: Fixed Sorted array to array.

**Pull request merged**